proxysql/datadog_checks/proxysql/queries.py

```diff
@@ -236,9 +236,9 @@
 
 STATS_MYSQL_BACKENDS = {
     'name': 'stats_mysql_backends',
-    'query': 'SELECT  hostgroup_id, status, COUNT(*) FROM runtime_mysql_servers GROUP BY hostgroup_id, status',
+    'query': 'SELECT  hostgroup, status, COUNT(*) FROM stats_mysql_connection_pool GROUP BY hostgroup, status',
     'columns': [
-        {'name': 'hostgroup_id', 'type': 'tag'},
+        {'name': 'hostgroup', 'type': 'tag'},
```

Comment:
If we change this value here, the tag will also change.

The name of the column here does not have to match the name in your query, we map them by position. It means you can update the query and keep `hostgroup_id` for this field so it won't affect the tag name in the end

Reply:
Ok thanks for this info ! Even if we don't have to match names we might still want to change tag name to `hostgroup` in this PR (same as in `connection_pool_metrics`)

Reply:
Sounds good to me!